docs/gke.md

```diff
@@ -0,0 +1,192 @@
+# Create Kubernetes cluster on Google Kubernetes Engine (GKE)
+
+## Prerequisites
+
+All commands from this guide can be run either in the **Google Cloud shell** or in **your local shell**.
+
+To use *Google Cloud shell*, you need nothing but a modern web browser.
+
+If you would like to use *your local shell*, install the following:
+
+1. [gcloud](https://cloud.google.com/sdk/docs/quickstarts). This tool is
+    part of the Google Cloud SDK. To install it, select your operating
+    system on the [official Google Cloud SDK documentation page](https://cloud.google.com/sdk/docs)
+    and then follow the instructions.
+
+2. [kubectl](https://cloud.google.com/kubernetes-engine/docs/quickstart#choosing_a_shell).
+    It is the Kubernetes command-line tool you will use to manage and deploy
+    applications. To install the tool, run the following command:
+
+    ``` {.bash data-prompt="$" }
+    $ gcloud auth login
+    $ gcloud components install kubectl
+    ```
+
+## Create and configure the GKE cluster
+
+You can configure the settings using the `gcloud` tool. You can run it either in
+the [Cloud Shell](https://cloud.google.com/shell/docs/quickstart) or in your
+local shell (if you have installed Google Cloud SDK locally on the previous
+step). The following command will create a cluster named `my-cluster-name`:
+
+``` {.bash data-prompt="$" }
+$ gcloud container clusters create my-cluster-name --project <project name> --zone us-central1-a --cluster-version {{ gkerecommended }} --machine-type n1-standard-4 --num-nodes=3
+```
+
+!!! note
+
+    You must edit the following command and other command-line statements to
+    replace the `<project name>` placeholder with your project name. You may
+    also be required to edit the *zone location*, which is set to `us-central1`
+    in the above example. Other parameters specify that we are creating a
+    cluster with 3 nodes and with machine type of 4 vCPUs.
+
+You may wait a few minutes for the cluster to be generated.
+
+???+ note "When the process is over, you can see it listed in the Google Cloud console"
+
+    Select *Kubernetes Engine* → *Clusters* in the left menu panel:
+
+    ![image](assets/images/gke-quickstart-cluster-connect.svg)
+
+Now you should configure the command-line access to your newly created cluster
```

Comment:
Why not making a bullet list? It's hard to follow what to do when scanning the doc. Users usually scan it, not read it carefully